orangecontrib/spectroscopy/widgets/preprocessors/normalize.py

```diff
@@ -27,6 +27,7 @@ class NormalizeEditor(BaseEditorOrange):
         ("Min-Max Normalization", Normalize.MinMax),
         ("Area Normalization", Normalize.Area),
         ("Attribute Normalization", Normalize.Attribute),
+        ("Standard Normal Variate (SNV)", Normalize.SNV),
```

Comment:
Is it redundant to say "(SNV)" here? I guess users will know (hopefully) what it is?